# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libfreehand/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libfreehand-0.1.2/src/lib/libfreehand_utils.cpp.vanilla	2024-05-22 10:36:18.866000000 +0200
+++ libfreehand-0.1.2/src/lib/libfreehand_utils.cpp	2024-05-22 10:36:34.216000000 +0200
@@ -162,7 +162,7 @@
   while (j < length)
   {
     UChar32 c;
-    U16_NEXT(s, j, length, c)
+    U16_NEXT(s, j, length, c);
     unsigned char outbuf[U8_MAX_LENGTH+1];
     int i = 0;
     U8_APPEND_UNSAFE(&outbuf[0], i, c);
